dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/fixturegenerator/elements/FixtureSectionBuilder.kt

```diff
@@ -13,8 +13,8 @@ class FixtureSectionBuilder(
     val elements: MutableList<FixtureGeneratorElement>,
 ) : FixtureGeneratorElement {
 
-    override val imports: Set<String> =
-        elements.foldRight(emptySet()) {
+    override val imports: Set<String>
+        get() = elements.foldRight(emptySet()) {
```

Comment:
Why this change?

Reply:
Previously, the imports were calculated just once,  during the initial instantiation of the FixtureSectionBuilder (Just the assignment). However, as this class is continuously updated, the imports must be re-calculated on every access (`get() =`)

Reply:
ok thank you